### DISCUSSION TITLE
July 22-July 28

### DISCUSSION THREAD

**Discussion by eeholmes** [UW-Upwelling-Project/Summer-2022#15] [Team Posts]
## William

* Keeping working on simulated data
* Determine how to detect changes in cycle
   * linear trend in the mean level
   * changing amplitude over time
   * breaks like Anthony showed

## Anthony
* Look at William's data to get better understanding of the 2 year cycle blip
* Try to replicate the Nature paper [on latitude ](https://www.nature.com/articles/nature14235)
    * latitude homogenization
* make map showing the lat/lon points that you are using

**Comment by eeholmes** [comment]
JULY 29 

**Anthony**

* simulated annual data -> only peak at 1 year
* tried to replicate the spatial homenity should increase
* Get the DAILY data to run in William's wavelet code

* correlation plot
* get the spatial average of the land temperature; SST 

SST, Daily Optimum Interpolation (OI), AVHRR Only, Version 2, Final, Global, 0.
25°, 1982-2020
https://coastwatch.pfeg.noaa.gov/erddap/griddap/ncdcOisst2Agg.html

**William**

Reading papers using wavelet analysis in oceanography

* Do your methods work on monthly data?
* Work with Anthony's data some.
Practical Guide to Wave
https://www.jstor.org/stable/26232616
Climate Signal Detection using wavelet transform: How to make sing
https://paos.colorado.edu/research/wavelets/bams_79_01_0061.pdf

![image](https://user-images.githubusercontent.com/2545978/181841318-6f11b856-b4bc-42ef-a643-e4445c110ef7.png)

![image](https://user-images.githubusercontent.com/2545978/181841737-6bf352c4-0be2-467b-83cb-3b597313e7e0.png)

**Comment by eeholmes** [comment]
@XuweiyiChen Post wavelet refs that were esp helpful here

**Comment by XuweiyiChen** [comment]
Lau, K. M., and Hengyi Weng. “Climate Signal Detection Using Wavelet Transform: How to Make a Time Series Sing.” Bulletin of the American Meteorological Society, vol. 76, no. 12, 1995, pp. 2391–402. JSTOR, http://www.jstor.org/stable/26232616. Accessed 31 Jul. 2022.

This one is particularly interesting. For more papers, please read the section 1 of this paper.

Torrence, Christopher and Gilbert P. Compo. “A Practical Guide to Wavelet Analysis.” Bulletin of the American Meteorological Society 79 (1998): 61-78.

I am working on this paper.

Gerald Kaiser. 1994. A friendly guide to wavelets. Birkhauser Boston Inc., USA.

For reference of details of wavelets, please refer to this paper.
